<ui version="4.0" >
 <class>Dialog</class>
 <widget class="QDialog" name="Dialog" >
  <property name="geometry" >
   <rect>
    <x>0</x>
    <y>0</y>
    <width>307</width>
    <height>142</height>
   </rect>
  </property>
  <property name="sizePolicy" >
   <sizepolicy vsizetype="Fixed" hsizetype="Fixed" >
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="minimumSize" >
   <size>
    <width>307</width>
    <height>140</height>
   </size>
  </property>
  <property name="maximumSize" >
   <size>
    <width>307</width>
    <height>142</height>
   </size>
  </property>
  <property name="windowTitle" >
   <string>Credentials Required</string>
  </property>
  <property name="windowIcon" >
   <iconset>../../../../SEA_Ant/ui/Skins/Default/icons/prod_color.png</iconset>
  </property>
  <widget class="QPushButton" name="upwOkayBtn" >
   <property name="geometry" >
    <rect>
     <x>130</x>
     <y>110</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text" >
    <string>Okay</string>
   </property>
   <property name="default" >
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QLineEdit" name="upwUsername" >
   <property name="geometry" >
    <rect>
     <x>90</x>
     <y>50</y>
     <width>201</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label" >
   <property name="geometry" >
    <rect>
     <x>20</x>
     <y>50</y>
     <width>61</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text" >
    <string>Username : </string>
   </property>
  </widget>
  <widget class="QLabel" name="label_2" >
   <property name="geometry" >
    <rect>
     <x>20</x>
     <y>80</y>
     <width>56</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text" >
    <string>Password : </string>
   </property>
  </widget>
  <widget class="QPushButton" name="upwDisconnectBtn" >
   <property name="geometry" >
    <rect>
     <x>220</x>
     <y>110</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text" >
    <string>Disconnect</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="upwPassword" >
   <property name="geometry" >
    <rect>
     <x>90</x>
     <y>80</y>
     <width>201</width>
     <height>20</height>
    </rect>
   </property>
   <property name="echoMode" >
    <enum>QLineEdit::Password</enum>
   </property>
  </widget>
  <widget class="QLabel" name="label_3" >
   <property name="geometry" >
    <rect>
     <x>20</x>
     <y>20</y>
     <width>71</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text" >
    <string>Connection :</string>
   </property>
  </widget>
  <widget class="QLabel" name="upwConnName" >
   <property name="geometry" >
    <rect>
     <x>110</x>
     <y>20</y>
     <width>191</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text" >
    <string>Unknown</string>
   </property>
  </widget>
 </widget>
 <tabstops>
  <tabstop>upwUsername</tabstop>
  <tabstop>upwPassword</tabstop>
  <tabstop>upwOkayBtn</tabstop>
  <tabstop>upwDisconnectBtn</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
